wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -442,10 +442,28 @@ <h3>Mixtures</h3>
     {% panel id="editor" heading="Page editor" %}
         <h3>Tabs</h3>
 
-        <div class="w-tabs" data-tabs>
-            <div role="tablist" class="w-tabs__list">
-                {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-1' title='Tab 1' %}
-                {% include 'wagtailadmin/shared/tabs/tab_nav_link.html' with tab_id='tab-2' title='Tab 2' %}
+        <div class="w-tabs" data-controller="w-tabs" data-action="popstate@window->w-tabs#loadHistory" data-w-tabs-selected-value="" data-w-tabs-selected-class="animate-in">
+            <div class="w-tabs__list" role="tablist" data-action="click->w-tabs#handleTabChange:prevent keydown.right->w-tabs#selectNext keydown.left->w-tabs#selectPrevious keydown.home->w-tabs#selectFirst keydown.end->w-tabs#selectLast">
+                <a id="tab-label-tab-1" href="#tab-tab-1" class="w-tabs__tab" role="tab" tabindex="-1" data-w-tabs-target="label">
+                    Tab 1
+                </a>
+                <a id="tab-label-tab-2" href="#tab-tab-2" class="w-tabs__tab" role="tab" data-w-tabs-target="label">
+                    Tab 2
+                </a>
+                <a id="tab-label-tab-3" href="#tab-tab-3" class="w-tabs__tab" role="tab" data-w-tabs-target="label">
+                    Tab 3
+                </a>
+            </div>
+            <div class="tab-content tab-content--comments-enabled">
+                <section id="tab-tab-1" class="w-tabs__panel " role="tabpanel" aria-labelledby="tab-label-tab-1" data-w-tabs-target="panel">
+                    all about tab 1
+                </section>
+                <section id="tab-tab-2" class="w-tabs__panel " role="tabpanel" aria-labelledby="tab-label-tab-2" data-w-tabs-target="panel">
+                    all about tab 2
+                </section>
+                <section id="tab-tab-3" class="w-tabs__panel " role="tabpanel" aria-labelledby="tab-label-tab-3" data-w-tabs-target="panel">
+                    all about tab 3
+                </section>
```

Comment:
I'd recommend against this change, sorry for the confusion but I meant that we should start adoption of the new data attributes in the shared includes. With this tabs being a good way to manually test.